pkgs/development/python-modules/jupyterhub-tmpauthenticator/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, jupyterhub
+}:
+
+buildPythonPackage rec {
+  pname = "jupyterhub-tmpauthenticator";
+  version = "0.6";
+  disabled = pythonOlder "3.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "064x1ypxwx1l270ic97p8czbzb7swl9758v40k3w2gaqf9762f0l";
+  };
+
+  propagatedBuildInputs = [ jupyterhub ];
+
+  doCheck = false;
```

Comment:
please give a comment as to why they were disabled.

I'm assuming either network or difficult to test. Either way, should give a reason.

Reply:
The package doesn't actually provide any tests so I figured there is no reason to run the check phase. I'll add a comment to the file.

Reply:
Comment added.